azure-pipelines.yml

```diff
@@ -122,7 +122,7 @@ jobs:
   - task: UsePythonVersion@0
     displayName: 'Use Python 3'
     inputs:
-      versionSpec: 3.x
+      versionSpec: 3.8
```

Comment:
The `dev` branch already has this change:

https://github.com/Azure/azure-cli/blob/d9afad75dcae385e24412ed753704ac430847879/azure-pipelines.yml#L125

Reply:
Ah. I see you are targeting the `release` branch. Remember to merge it back to `dev` later.